src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -269,14 +254,14 @@ const TfEditor: React.FC<TfEditorProps> = (props) => {
     () => d3.scaleLinear().domain([plotMin, plotMax]).range([0, innerWidth]),
     [innerWidth, plotMin, plotMax]
   );
-  const plotMinU8 = useMemo(() => absoluteToBin(plotMin, histogram), [plotMin, histogram]);
-  const plotMaxU8 = useMemo(() => absoluteToBin(plotMax, histogram), [plotMax, histogram]);
+  const plotMinU8 = useMemo(() => histogram.findFractionalBinOfValue(plotMin), [plotMin, histogram]);
+  const plotMaxU8 = useMemo(() => histogram.findFractionalBinOfValue(plotMax), [plotMax, histogram]);
```

Comment:
why do we still need plotMinU8 and plotMaxU8?

Reply:
The histogram is still plotted in a domain relative to bin indices. I believe this gets overhauled in #437 by @frasercl